signing_service/signing_service/signing_service.py

```diff
@@ -268,6 +279,23 @@ def _handle_connection(
             # If received frame is correct, validate transaction.
             else:
                 golem_message_response = self._get_signed_transaction(middleman_message.payload)
+                if isinstance(golem_message_response, SignedTransaction):
+                    transaction_sum_combined = self.signing_service_daily_transaction_sum_so_far + middleman_message.payload.value
+                    if transaction_sum_combined > MAXIMUM_DAILY_THRESHOLD:
+                        logger.warning(
+                            f'Signing Service is unable to transact more then {MAXIMUM_DAILY_THRESHOLD} GNTB today.'
```

Comment:
Consider add client_key to identify which transaction was rejected

Reply:
Done